src/transformers/models/auto/processing_auto.py

```diff
@@ -40,7 +40,7 @@
 PROCESSOR_MAPPING_NAMES = OrderedDict(
     [
         ("clip", "CLIPProcessor"),
-        ("donut", "DonutProcessor"),
+        ("donut-swin", "DonutProcessor"),
```

Comment:
Don't think this makes sense as donut-swin is just the vision encoder, so it doesn't need a processor. However I wonder how people can leverage the AutoProcessor API with donut checkpoints

Reply:
@NielsRogge I can see `("swin", "ViTFeatureExtractor")` in `FEATURE_EXTRACTOR_MAPPING_NAMES`. Can I add `("donut-swin", "ViTFeatureExtractor")`.

I agree that we should remove `("donut-swin", "DonutProcessor"),` from `PROCESSOR_MAPPING_NAMES` here.

Reply:
Regarding Auto Mappings for Encoder-Decoder like models, it will need another thread to discuss.

Reply:
Let me check if currently this "donut" in the mappings will work.

Reply:
@NielsRogge 

The checkpoint `naver-clova-ix/donut-base` has `preprocessor_config` with `feature_extractor_type` and `processor_class` being specified, and this is sufficient for AutoProcessor/AutoFeatureExtractor work with it

https://huggingface.co/naver-clova-ix/donut-base/blob/main/preprocessor_config.json

Reply:
I removed `("donut-swin", "DonutProcessor")` as suggested by @NielsRogge

Reply:
Thanks!